src/main/menu.js

```diff
@@ -207,9 +207,10 @@ export function setupMenu(store) {
             ...(isWindows()
               ? [
                   {
-                    label: 'Auto-hide Menubar',
+                    label: 'Hide/Show Menubar',
                     type: 'checkbox',
                     checked: !menubarVisible,
+                    accelerator: 'CmdOrCtrl+Shift+H',
```

Comment:
This is nice, but a couple of things that would be nice to iron out:

 * Why not Alt? Alt is generally accepted (and programmed) in windows to show file/menu bars when they're hidden (as mentioned [here](https://www.computerhope.com/issues/ch000973.htm) for example).
 * If we have to stick to a different accelerator for some reason, is `H` the best key? It could be for **H**ide, sure, but there may be many things in the interface that we want to hide or show. I would suggest `M` personally, for menu bar, but now I'm not sure what other keys we could potentially be stepping on.

What's your opinion @sallar?